components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIClientGenerationManager.java

```diff
@@ -88,65 +88,24 @@ public APIClientGenerationManager() {
      * @param apiVersion  version of the API
      * @param apiProvider provider of the API
      * @param loggedInUsername username of the currently logged in user
+     * @param swaggerAPIDefinition Swagger Definition of the API
      * @return a map containing the zip file name and its' temporary location until it is downloaded
      * @throws APIClientGenerationException if failed to generate the SDK
      */
     public Map<String, String> generateSDK(String sdkLanguage, String apiName, String apiVersion, String apiProvider,
-                                           String loggedInUsername)
+                                           String loggedInUsername, String swaggerAPIDefinition)
```

Comment:
Can we review method parameters and remove unwanted ones?

We don't need 'loggedInUsername' I think

Reply:
Fixed